spinn_front_end_common/interface/front_end_common_interface_functions.py

```diff
@@ -498,21 +504,22 @@ def start_all_cores(self, executable_targets, app_id, no_full_runs):
                     "Only {} of {} processors started:{}"
                     .format(processors_running, total_processors, break_down))
 
-    def wait_for_execution_to_complete(
-            self, executable_targets, app_id, runtime, time_scaling):
+    def wait_for_execution_to_complete(self, executable_targets, app_id,
+        runtime, time_scaling, no_full_runs):
         """
 
         :param executable_targets:
         :param app_id:
         :param runtime:
         :param time_scaling:
+        :param no_full_runs: the number of runs been done between setup and end
         :return:
         """
 
         total_processors = executable_targets.total_processors
         all_core_subsets = executable_targets.all_core_subsets
 
-        time_to_wait = ((runtime * time_scaling) / 1000.0) + 1.0
+        time_to_wait = ((runtime * time_scaling) / 1000.0) * 0.9
```

Comment:
I dont understand why theres a * 0.9 vs the +1.0 that was there orginally. Can you shed some light on this please.